src/viam/components/encoder/encoder.py

```diff
@@ -64,23 +64,24 @@ async def get_position(
         Report the position of the encoder.
         The value returned is the current position in terms of it's ``position_type``.
         The position will be either in relative units (ticks away from a zero position) for
-        ``PositionType.TICKS`` or absolute units (degrees along a circle) for ``PositionType.DEGREES``.
+        ``PositionType.POSITION_TYPE_TICKS_COUNT`` or absolute units (degrees along a circle)
+        for ``PositionType.POSITION_TYPE_ANGLE_DEGREES``.
 
         ::
 
             my_encoder = Encoder.from_robot(robot=robot, name='my_encoder')
 
             # Get the position of the encoder in ticks
-            position = await my_encoder.get_position(encoder.PositionTypeTicks)
+            position = await my_encoder.get_position(PositionType.POSITION_TYPE_TICKS_COUNT)
             print("The encoder position is currently ", position[0], position[1])
 
         Args:
-            position_type (PositionType.ValueType): The desired output type of the position.
```

Comment:
This technically needs to be `PositionType.ValueType` because of the way protobuf implements these types of enumerations. If this links, then it should link to the `.ValueType`

Reply:
Ahh ok-- the link wasn't being generated by the parser. I'll try adding it manually?

Reply:
Ohh interesting. Ok for linking purposes I think we can keep this as just `PositionType` then. The actual underlying type is a property of `PositionType`, so it should be straightforward. And since the actual method signature has the correct type, typechecking should work just fine. 

Up to y'all as this only affects docs.

Reply:
That's good with me, I figured it was straightforward as well